src/Nebulas/Address.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Address.h"
+#include "../Hash.h"
+#include "../Base58.h"
+#include "../HexCoding.h"
+
+using namespace TW::Nebulas;
+
+bool Address::isValid(const std::string& string) {
+    auto data = Base58::bitcoin.decode(string);
+    if (data.size() != Address::size) {
+        return false;
+    }
+
+    if(data[0] != Address::AddressPrefix) {
+        return false;
+    }
+    if (data[1] != Address::NormalType && data[1] != Address::ContractType) {
+        return false;
+    }
+
+    Data content(data.begin(), data.begin()+22);
```

Comment:
```suggestion
    Data content(data.begin(), data.begin() + 22);
```

Reply:
I have used the Clang-format for this file.